google-cloud-speech/google-cloud-speech.gemspec

```diff
@@ -12,6 +12,12 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://github.com/googleapis/googleapis"
   gem.license       = "Apache-2.0"
 
+  gem.post_install_message =
+    "This release, 0.30.0, introduces breaking changes relative to the "\
```

Comment:
I would recommend future-proofing this sentence with: "The 0.30.0 release introduced ..."